test/test_torch.py

```diff
@@ -9705,6 +9705,12 @@ def test_contains(self):
         val += 10
         self.assertEqual(val in x, False)
 
+        with self.assertRaisesRegex(
+            RuntimeError,
+            'Tensor.__contains__ only supports scalars'):
+            "foo" in x
```

Comment:
I'm not sure the second line `[1, 2] in x` is run  -- this should go under two different assertRaisesRegex expressions

Reply:
You said it! My bad...